content/guides/digging-deeper/mailer.md

```diff
@@ -724,20 +724,30 @@ When using mailer classes, you can call the `preview` method directly on the cla
 ## Email traps
 AdonisJS allows you to trap the outgoing emails instead of sending them to the actual recipient. The traps are usually helpful when writing tests. 
 
-All you need to do is call the `Mail.trap` method and pass it the callback to receive the outgoing message.
+All you need to do is call the `Mail.trap` method before you trigger the message and pass it the callback to receive the outgoing message. Wrap your assertions in a try/catch block and provide your errors to the `done()` function of your test runner to report the actual message errors.
```

Comment:
I think this should be the obvious assumption vs documentation clearing out the flow of the execution of the code. This will make the documentation verbose.

I will skip this PR for now

Reply:
No worries, thought it might help newcomers. It had me stumped for a while until you responded on the discussion.

Reply:
I agree. But, I have to maintain some balance in the docs. Explaining every little thing will make docs bloated and harder to standout main information.

I do appreciate you taking time. Also, I do encourage PRs to improve the docs. But again, at times have to skip some :)